src/components/switch/switch.component.tsx

```diff
@@ -139,7 +144,9 @@ export const Switch = React.forwardRef(
       error,
       warning,
       info,
-      useValidationIcon: !shouldValidationBeOnLabel && !disabled,
+      useValidationIcon: !validationRedesignOptIn
```

Comment:
**suggestion** (non-blocking) - I personally think this is much easier to understand when written as `useValidationIcon: !validationRedesignOptIn && !shouldValidationBeOnabel && !disabled`

Reply:
Further thoughts on this - I could be barking up completely the wrong tree but I thought worth mentioning as it might be helpful:

As far as just this property is concerned, it would be neater if `useValidationIcon` stayed the same as before but the definition of `shouldValidationBeOnLabel` changed so that it took `validationRedesignOptIn` into account.

Then I realised that `shouldValidationBeOnLabel` is also used in `validationOnLabel` so any such change would also affect the value of that prop that's passed to `CheckableInput`.

BUT it's possible that doing this will actually fix `validationOnLabel` so that it's doing what we want in `CheckableInput` and doesn't require messing with `FormField` which, as I observed above, is causing some issues as it stands now.

Not making this an "official" comment but maybe there's something useful in there, even if unlikely 😅

Reply:
Good shout. I'll make sure it's changed to `useValidationIcon: !validationRedesignOptIn && !shouldValidationBeOnabel && !disabled`